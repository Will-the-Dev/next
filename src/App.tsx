import React from 'react';
import { Provider } from 'react-redux';

import './App.css';

import { Main } from './pages';
import { store } from './store';

function App() {
    return (
        <Provider store={store}>
            <Main />
        </Provider>
    );
}

export default App;
