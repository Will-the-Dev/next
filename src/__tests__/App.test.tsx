import React from 'react';
import { render } from '@testing-library/react';

import App from '../App';

it('should render component to match snapshot', () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
});
