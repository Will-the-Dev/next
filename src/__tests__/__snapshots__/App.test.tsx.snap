// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component to match snapshot 1`] = `
<DocumentFragment>
  .c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
  width: 100%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  height: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 60px auto;
  max-width: 1200px;
  padding: 0px 14px;
  width: calc(100% - 28px);
}

.c2 > * {
  margin: 16px;
}

.c0 {
  margin: 30px auto;
  max-width: 570px;
  z-index: 1;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 div:first-child {
  margin-right: 15px;
}

<div>
    <div
      class="c0"
    >
      <form
        data-testid="form"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4 MuiTypography-colorInherit MuiTypography-alignLeft"
        >
          Search for song, artist or album.
        </h4>
        <br />
        <div
          class="c1"
        >
          <div
            class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <div
              class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorSecondary MuiOutlinedInput-colorSecondary MuiInputBase-fullWidth MuiInputBase-formControl"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputTypeSearch"
                placeholder="search"
                type="search"
                value=""
              />
              <fieldset
                aria-hidden="true"
                class="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
                style="padding-left: 8px;"
              >
                <legend
                  class="PrivateNotchedOutline-legend-2"
                  style="width: 0.01px;"
                >
                  <span>
                    â€‹
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained"
            tabindex="0"
            type="submit"
          >
            <span
              class="MuiButton-label"
            >
              Submit
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </form>
    </div>
    <div
      class="c2"
    />
    <div
      class="c3"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg"
      />
    </div>
  </div>
</DocumentFragment>
`;
